lib/client.js

```diff
@@ -471,7 +471,7 @@ AT.prototype.linkClick = function(route){
     var firstVisibleInput = _.find(this.getFields(), function(f){
       return _.contains(f.visible, route);
     });
-    if (firstVisibleInput) {
+    if (AccountsTemplates.options.focusFirstInput && firstVisibleInput) {
```

Comment:
at this point I'd move the first condition before `var firstVisibleInput` to save a bit of time in case the option is not set...